subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/FindBugs.java

```diff
@@ -328,12 +329,26 @@ public FileTree getSource() {
     @SkipWhenEmpty
     @PathSensitive(PathSensitivity.RELATIVE)
     @InputFiles
+    public FileCollection getCandidateClasses() {
```

Comment:
A javadoc would be nice here.
We have `getCandidateClassFiles()` on the `Test` task. Should we use the same name here?
Should we make this method `protected` so we somewhat hide it for consumers of the class?